lib/visual-website-optimizer/index.js

```diff
@@ -62,6 +106,28 @@ VWO.prototype.replay = function(){
  * @return {Object}
  */
 
+function rootExperiments(fn){
+  enqueue(function(){
+    var data = {};
+    var expIds = window._vwo_exp_ids;
+    if (!expIds) return fn();
+    each(expIds, function(expId){
+      var varName = variation(expId);
+      if (varName) data[expId] = varName;
```

Comment:
Is there a reason why line 138 and this line are slightly different? (This is missing the `'Experiment ' +`)